packages/govern-server/src/core/data/TheGraphClient.ts

```diff
@@ -0,0 +1,71 @@
+import fetch from 'isomorphic-unfetch'
+import { Client } from '@urql/core'
+import { DocumentNode } from 'graphql'
+import { ErrorConnection } from '../errors'
+import { QueryResult } from '../types'
+
+type TheGraphClientOptions = {
+  verbose?: boolean
+}
+
+class TheGraphClient {
+  #client: Client
+  #verbose: boolean
+
+  constructor(
```

Comment:
It isn't required to have it on a new line here cause it does not exceed the line-length.